data-assets/rs-namespace/deploy/acm_right_sizing_grafana_dashboard.yaml

```diff
@@ -739,7 +817,7 @@ data:
             "sortBy": [
               {
                 "desc": false,
-                "displayName": "CPU Utilization %"
+                "displayName": "Namespace"
```

Comment:
Is there any reason for this change? If not lets keep sort order as `CPU Utilization %` ?